tools/converter/common/src/main/java/org/wso2/ei/tools/converter/common/ballerinahelper/Message.java

```diff
@@ -0,0 +1,114 @@
+                .addTypes(org.wso2.ei.tools.converter.common.util.Constant.BLANG_TYPE_STRING); //type of the variable
+        ballerinaASTModelBuilder.addReturnTypes();
+    }
+
+    /**
+     * Get payload
+     *
+     * @param ballerinaASTModelBuilder
+     * @param parameters
+     */
+    public static void getPayload(BallerinaASTModelBuilder ballerinaASTModelBuilder, Map<String, Object> parameters) {
+        ballerinaASTModelBuilder.addTypes((String) parameters.get(Constant.TYPE)); //type of the variable
+        ballerinaASTModelBuilder
+                .createNameReference(org.wso2.ei.tools.converter.common.util.Constant.BLANG_PKG_MESSAGES,
+                        (String) parameters.get(Constant.FUNCTION_NAME));
+        ballerinaASTModelBuilder.createSimpleVarRefExpr();
+        ballerinaASTModelBuilder.startExprList();
+        ballerinaASTModelBuilder.createNameReference(null, (String) parameters.get(Constant.INBOUND_MSG));
+        ballerinaASTModelBuilder.createSimpleVarRefExpr();
+        ballerinaASTModelBuilder.endExprList(1);
+        ballerinaASTModelBuilder.addFunctionInvocationExpression(true);
+        ballerinaASTModelBuilder
+                .createVariable((String) parameters.get(Constant.VARIABLE_NAME), true); //name of the variable
+        ballerinaASTModelBuilder.addTypes((String) parameters.get(Constant.TYPE)); //type of the variable
+        ballerinaASTModelBuilder.addReturnTypes();
+    }
+
+    /**
+     * Set the payload of a message
+     */
+    public static void setPayload(BallerinaASTModelBuilder ballerinaASTModelBuilder, Map<String, Object> parameters) {
+
+        if (Constant.JSON.equals((String) parameters.get(Constant.TYPE))) {
+            ballerinaASTModelBuilder.addComment(Constant.BLANG_COMMENT_JSON);
+            ballerinaASTModelBuilder.addTypes(Constant.BLANG_TYPE_JSON); //type of the variable
+            ballerinaASTModelBuilder.createStringLiteral((String) parameters.get(Constant.FORMAT));
+            ballerinaASTModelBuilder.createVariable((String) parameters.get(Constant.PAYLOAD_VAR_NAME), true); //name of
+            // the variable
+            ballerinaASTModelBuilder.createNameReference(Constant.BLANG_PKG_MESSAGES, Constant.BLANG_SET_JSON_PAYLOAD);
+
+        } else if (Constant.XML.equals((String) parameters.get(Constant.TYPE))) {
+            ballerinaASTModelBuilder.addTypes(Constant.BLANG_TYPE_XML); //type of the variable
+            ballerinaASTModelBuilder.addComment("//TODO: Change the double quotes to back tick. ");
+            ballerinaASTModelBuilder.createXMLLiteral((String) parameters.get(Constant.FORMAT));
+            ballerinaASTModelBuilder.createVariable((String) parameters.get(Constant.PAYLOAD_VAR_NAME), true); //name of
+            // the variable
+            ballerinaASTModelBuilder.createNameReference(Constant.BLANG_PKG_MESSAGES, Constant.BLANG_SET_XML_PAYLOAD);
+        }/* else if (Constant.STRING.equals((String) parameters.get(Constant.TYPE))) {
```

Comment:
Shall we remove this commented line ?

Reply:
they were removed in a later commit. https://github.com/wso2/product-ei/commit/8eed353ae62790097571f73501170663b0bdb8f9#diff-609b8bd6feb33c1f80efc77990291592L85